README.md

```diff
@@ -66,17 +58,21 @@ sudo apt clean && sudo apt update
 ```
```

Comment:
This seems a bit cargo-cultish.

What does this solve exactly? The [changed key](https://answers.ros.org/question/325039) problem?

Reply:
I would remove it.

Or if @chcorbato wants to keep it, don't just describe it as "running into an issue", but be clear about what the error is. Otherwise ppl will just copy-paste magic incantations and hope it will work for them.